framework/wazuh/core/config/models/server.py

```diff
@@ -281,17 +281,37 @@ class JWTConfig(WazuhConfigBaseModel, ValidateFilePathMixin):
     ----------
     private_key : str
         The path to the private JTW key file.
-    public_key : str
-        The path to the public JTW key file.
+    _public_key : str
+        The public JTW key.
```

Comment:
Same in the `private_key` description

```suggestion
        The public JWT key.
```

Reply:
Changed in 77dfbbd6ab6a16df3f4a0984fbbb3640d6cc1164.